### PR TITLE
Add document for CSV result visualization

### PR DESCRIPTION
**Pull request by thejasbabu**
Signed-off-by: thejas <thejasb99@gmail.com>

### CODE REVIEW

File: src/data/markdown/docs/01 guides/03 Results visualization/CSV.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Or if you want to get the result gzipped, like this:
+
+<div class="code-group" data-props='{"labels": ["CLI"]}'>
+
+```shell
+$ k6 run --out csv=my_test_result.gz script.js
+```
+
+</div>
```
Comment (MStoykov) [inline src/data/markdown/docs/01 guides/03 Results visualization/CSV.md:24]: This,technically, will be available after 0.28.0 is released, maybe we can add this as a note somewhere

File: src/data/markdown/docs/01 guides/03 Results visualization/CSV.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+checks,1595325561,0.000000,response body,,,,,,,default,,,,,
+data_sent,1595325561,76.000000,,,,,,,,default,,,,,
+data_received,1595325561,11045.000000,,,,,,,,default,,,,,
+iteration_duration,1595325561,1449.049580,,,,,,,,default,,,,,
+iterations,1595325561,1.000000,,,,,,,,default,,,,,
+```
+
+</div>
+
+Each entry in the report represents a metric `metric_name` along with it's value `metric_value` at time `timestamp`. If error, then the `error` along with the `error_code` is captured.
```
Comment (MStoykov) [inline src/data/markdown/docs/01 guides/03 Results visualization/CSV.md:54]: Maybe link to https://k6.io/docs/javascript-api/error-codes as well ?

File: src/data/markdown/docs/01 guides/03 Results visualization/CSV.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+title: 'CSV'
+excerpt: ''
+---
+
+You can also make k6 output detailed statistics in a CSV format by using the `--out/-o` option for `k6 run`, like this:
```
Comment (imiric) [inline src/data/markdown/docs/01 guides/03 Results visualization/CSV.md:6]:
```suggestion
You can also make k6 output detailed statistics in a CSV format by using the `--out`/`-o` option for `k6 run`, like this:
```

File: src/data/markdown/docs/01 guides/03 Results visualization/CSV.md (hunk 2 above)
Comment (imiric) [inline src/data/markdown/docs/01 guides/03 Results visualization/CSV.md:54]:
```suggestion
Each entry in the report represents a metric `metric_name` along with its value `metric_value` at time `timestamp`. If there's an error, then the `error` and `error_code` fields will be populated.
```

File: src/data/markdown/docs/01 guides/03 Results visualization/CSV.md (hunk 1 above)
Comment (imiric) [reply src/data/markdown/docs/01 guides/03 Results visualization/CSV.md:24]: Yeah, maybe we could comment it out for now, and uncomment it once 0.28.0 is released.

Comment (thejasbabu) [reply src/data/markdown/docs/01 guides/03 Results visualization/CSV.md:24]: Do you want me to comment the entire document? Or should I leave a note?

File: src/data/markdown/docs/01 guides/03 Results visualization/CSV.md (hunk 2 above)
Comment (thejasbabu) [reply src/data/markdown/docs/01 guides/03 Results visualization/CSV.md]: Done

File: src/data/markdown/docs/01 guides/03 Results visualization/CSV.md (hunk 1 above)
Comment (MStoykov) [reply src/data/markdown/docs/01 guides/03 Results visualization/CSV.md:24]: Sorry ... I have apparently forgotten to answer you are haven't seen the email. 
This although makes this easier as ... well we released 0.28.0 :D

**Review by MStoykov — APPROVED** [review]

**Comment by ppcano** [comment]
@thejasbabu , thank you for your contribution to the project! 👏

Merging.

**Pull request merged**